### PR TITLE
fix insert ci sometimes fail

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

File: tests/tck/features/insert/InsertIfNotExists.feature [hunk 1]
```diff
@@ -117,7 +117,7 @@ Feature: Insert vertex and edge with if not exists
       | like | src   | dst     |
       | 100  | "Tom" | "Conan" |
     # insert multi vertex multi tags
-    When executing query:
```
Comment (Shylock-Hg) [inline tests/tck/features/insert/InsertIfNotExists.feature:120]: What's the error?

Comment (nevermore3) [reply tests/tck/features/insert/InsertIfNotExists.feature:120]: Insert student for the first time

Comment (Shylock-Hg) [reply tests/tck/features/insert/InsertIfNotExists.feature:120]: I see in previous stage contains `try to execute query` already. So I don't add it is the resolution.

Comment (nevermore3) [reply tests/tck/features/insert/InsertIfNotExists.feature:120]: fix

Comment (Shylock-Hg) [reply tests/tck/features/insert/InsertIfNotExists.feature]: And it's same to the other modification.

**Review by Shylock-Hg — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Pull request merged**